src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -703,7 +703,7 @@ private List<ProjectGraphEntryPoint> GetProjectGraphEntryPoints(string entryProj
         /// <param name="projectInnerNodes">An <see cref="IReadOnlyDictionary{NuGetFramework,ProjectInstance} "/> containing the projects by their target framework.</param>
         /// <param name="isCpvmEnabled">A flag that is true if the Central Package Management was enabled.</param>
         /// <returns>A <see cref="List{TargetFrameworkInformation}" /> containing the target framework information for the specified project.</returns>
-        internal static List<TargetFrameworkInformation> GetTargetFrameworkInfos(IReadOnlyDictionary<string, IMSBuildProject> projectInnerNodes, bool isCpvmEnabled)
+        internal static List<TargetFrameworkInformation> GetTargetFrameworkInfos(IReadOnlyDictionary<string, IMSBuildProject> projectInnerNodes, bool isCpvmEnabled, bool isPruningEnabledGlobally)
```

Comment:
Maybe a comment for param isPruningEnabledGlobally indicating it can be overridden by an MSBuild property?